### PR TITLE
Add no-response action

### PR DESCRIPTION
**Pull request by nirinchev**

### CODE REVIEW

**Review by bmunkholm — APPROVED** [review]
Thanks!

File: synced-files/.github/workflows/no-response.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```
Comment (bmunkholm) [inline synced-files/.github/workflows/no-response.yml:13]: BTW, we should only need to run this once a day? @nirinchev

File: synced-files/.github/workflows/no-response.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
```
Comment (bmunkholm) [inline synced-files/.github/workflows/no-response.yml:19]: Should we also use a fixed commit here?

Comment (nirinchev) [reply synced-files/.github/workflows/no-response.yml:19]: Unfortunately, that action is packaged weirdly so only tagged releases have the actual executable code built.

File: synced-files/.github/workflows/no-response.yml (hunk 1 above)
Comment (nirinchev) [reply synced-files/.github/workflows/no-response.yml:13]: Yeah, doesn't matter too much how often we do it - the action itself is super cheap but we can tweak the schedule to once a day.

**Pull request merged**